src/views/Send.vue

```diff
@@ -117,42 +126,44 @@
     <div class="send-confirm wrapper form text-center" v-if="showConfirm">
       <div class="wrapper_top form">
         <div class="form-group">
-          <label> Send <span class="text-muted">(INCL FEES)</span> </label>
+          <label>
+            Send <span v-if="includeFees" class="text-muted">(INCL FEES)</span>
+          </label>
           <p class="confirm-value" :style="getAssetColorStyle(asset)">
             {{ amountToSend }} {{ asset }}
           </p>
+          <p v-if="!includeFees">~{{ totalFee }} ETH FEES</p>
```

Comment:
This is kind of strange. Why only BTC will show fees as included? It's all the same model
The fee is separate from the amount you send

Reply:
Hi @monokh I asked to Thessy and the design proposal shows the fees included only for BTC 
<img width="260" alt="Screen Shot 2020-12-01 at 9 40 32 AM" src="https://user-images.githubusercontent.com/1889891/100754618-497fa300-33b9-11eb-8f3e-92836a42fb91.png">

Reply:
I think she may be confused something about that... There is no difference in how those are handled, maybe she thought in BTC, fees are taken off what you send.